web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -32,6 +33,7 @@
   let isShowDeleteConfirmation = false;
   let addToSharedAlbum = true;
   let shouldPlayMotionPhoto = false;
+  let isShowProfileImageCrop = false;
```

Comment:
I wonder if there's a better way to handle this. A Boolean for each modal seems verbose and strange.

Reply:
Well i think its out of scope of this pr to change that, maybe in a future pr.